lib/modules/commentDepth.js

```diff
@@ -39,9 +44,14 @@ module.options = {
 		}, {
 			name: 'commentDepth',
 			type: 'text',
+			value: '4',
```

Comment:
I think all text option default values should be strings (until we have a number type) to emphasize that you should use `parseInt`/`parseFloat` since it _could_ be saved as a string